languages/go/internal/ffi/native/polar.h

```diff
@@ -13,27 +13,29 @@ typedef struct polar_Query polar_Query;
 
 const char *polar_get_error(void);
 
-polar_Polar *polar_new(void);
+struct polar_Polar *polar_new(void);
```

Comment:
what's up with the changes in this file?

Reply:
not sure, I thought they were generated by the polar-crossgen. @samscott89 do you know if this was supposed to happen?

Reply:
That shouldn't be a side effect of polar-crossgen, no... Although I have also started to see these popping up in my `polar-c-api/polar.h` file lately... I assumed it was something to do with Rust/cbindgen versions changing

Reply:
ok replaced it with `polar.h` from `main`

Reply:
Yeah its probably just an update to cbindgen/rust. It also generates typedefs so it's valid with or without saying `struct`. Fine to just commit whatever comes out.